nadlogar/problems/models.py

```diff
@@ -239,3 +239,149 @@ def generate(self):
         razstavljen = sympy.factor(izraz)
 
         return {"izraz": sympy.latex(izraz), "razstavljen": sympy.latex(razstavljen)}
+
+
+class ElementiMnozice(Problem):
+    """Problem za izpis elementov množice iz podanega predpisa"""
+
+    linearna_kombinacija = models.BooleanField(
+        "linearna kombinacija",
+        help_text="Ali naj naloga vsebuje linearno kombinacijo?",
+        choices=[(True, "Da"), (False, "Ne")],
+    )
+
+    class Meta:
+        verbose_name = "elementi množice"
+
+    def generate(self):
+        pogoj = random.choice(["|", "<", "<="])
```

Comment:
Tak, enterprise način (tm) tukaj bi bil, da bi definiral class-level konstantno spremenljivko `POGOJI = ('|' ...)`. Tuple namesto lista, da sporočiš, da je ta senzma konstanta. pod tem se definira še mapping `pogoj_latex`, (torej `POGOJ_LATEX` po novem). Point je, da je logika čim krajša, podatke se pa izloči posebej.

Reply:
Ali bi bilo smiselno `generate` razbiti na dva dela: `generate`, ki ustvari slovar osnovnih podatkov, in npr. `display`, ki naredi končen prikaz?